src/component/actions/SetContactAttrib/SetContactAttribFormHelper.test.ts

```diff
@@ -18,51 +21,70 @@ const setContactNameAction = createSetContactNameAction();
 const setContactLanguageAction = createSetContactLanguageAction();
 const setContactChannelAction = createSetContactChannelAction();
 
-const setContactFieldFormState = formHelper.actionToState(
-    setContactFieldAction,
+const setContactFieldFormState = formHelper.initializeForm(
+    {
+        originalNode: null,
```

Comment:
tests are the only place where originalNode can be null, this is why the data model requires it